src/app/src/components/FilterSidebar.jsx

```diff
@@ -153,7 +153,14 @@ class FilterSidebar extends Component {
         const insetComponent = (() => {
             switch (activeFilterSidebarTab) {
                 case filterSidebarTabsEnum.guide:
-                    return <FilterSidebarGuideTab />;
+                    return (
```

Comment:
The ordering of the tabs without vector tiles is: Guide, **Search**, Facilities.
The ordering with vector tiles is: **Facilities**, Search, Guide.

(bold indicates default selection).

Is this change in order expected?

Reply:
Yeah, we changed the tab order when vector tiles are active in https://github.com/open-apparel-registry/open-apparel-registry/pull/771 because we can now start fetching all the facilities by default in the vector tile version.